sigstore/_cli.py

```diff
@@ -275,9 +276,16 @@ def _verify(
     certificate_path: BinaryIO,
     signature_path: BinaryIO,
     cert_email: Optional[str],
+    cert_oidc_issuer: Optional[str],
     rekor_url: str,
     staging: bool,
 ) -> None:
+    if cert_email and not cert_oidc_issuer:
+        click.echo(
+            "--cert-oidc-issuer is required if --cert-email is provided", err=True
+        )
+        raise click.Abort
```

Comment:
We should confirm that `cosign` is actually enforcing this currently. If not, perhaps we should just warn for the time being and allow either/or?

Reply:
I'll just remove that check. The linked issue is still open and has no PR associated with it.